Makefile

```
@@ -331,9 +338,9 @@ mint-c: build-rust build-c-self-contracts
 # Side A merges (#234, #241, #272, feat/php-kit) and toolchain CI integration
 # (#245 in flight, #274 follow-up).
 .PHONY: all-mint
-all-mint: mint-rust mint-go mint-cpp mint-ts mint-csharp mint-java mint-python mint-c mint-zig
+all-mint: mint-rust mint-go mint-cpp mint-ts mint-csharp mint-java mint-python mint-c mint-ruby mint-zig
```

checkmake (0.3.2)

[warning] Target body for "all-mint" exceeds allowed length of 5 lines (12).

(maxbodylength)